schemas/authenticate-hawk-request.yml

```diff
@@ -42,13 +42,19 @@ properties:
       string that follows the HTTP method.
       **Note,** order of querystring elements is important.
   host:
-    type:                 string
```

Comment:
How does removing this help?

Once the type generation code sees `type: string` it could ignore all the other properties... as all of the `oneOf/anyOf` options are unimportant... If we have `type: string` we make it a string (unless we have `type: string, format: date-time`)

But `type: string, oneOf: ...` should just be a string... wouldn't that be better to fix in the codegen part?